apps/mobile/src/components/action-bar/action-bar.tsx

```diff
@@ -92,37 +92,60 @@ interface ActionBarContainerProps {
   children: ReactNode;
 }
 
-function ActionBarContainer({ children }: ActionBarContainerProps) {
+function ActionBarBlurView({ children }: { children: ReactNode }) {
   const { bottom } = useSafeAreaInsets();
   const bottomOffset = getActionBarBottomOffset(bottom);
   const theme = useTheme<Theme>();
+  const { themeDerivedFromThemePreference } = useSettings();
+
+  const styles = {
+    position: 'absolute',
+    bottom: bottomOffset,
+    flexDirection: 'row',
+    alignSelf: 'center',
+    gap: theme.spacing[6],
+    height: actionBarHeight,
+    marginHorizontal: theme.spacing[5],
+    paddingHorizontal: theme.spacing[6],
+    paddingVertical: theme.spacing[2],
+    overflow: 'hidden',
+    borderRadius: theme.borderRadii['sm'],
+    borderColor: theme.colors['ink.border-transparent'],
+    borderWidth: 1,
+  } as const;
+
+  switch (Platform.OS) {
```

Comment:
`Platform.select` you used in [create wallet](https://github.com/leather-io/mono/blob/42eb396a3ba78634465adfeec8c2c33db8e6f115/apps/mobile/src/app/create-new-wallet.tsx#L30) seemed much less verbose. Can we restort to that instead adding a component?

Reply:
No you don't 😛